oauth2client/contrib/gce.py

```diff
@@ -22,11 +22,10 @@
 import warnings
 
 import httplib2
+import datetime
```

Comment:
Import order should be change here. Imports should be ordered first in sections (std lib then 3rd party then local imports) and then alphabetical within sections. (i.e. `datetime` is in the standard library but `httplib2` is 3rd party)